src/_components/guides/category.rb

```diff
@@ -0,0 +1,5 @@
+class Guides::Category < Bridgetown::Component
+  def initialize(resources:, title:, css_classes: "", position: nil)
```

Comment:
Depending of how you are using these positions, you may not need to test _if position.present? _
```suggestion
  def initialize(resources:, title:, css_classes: "", position: 0)
```

Reply:
This won't do here as I use the absence of this data-attribute to manage some layouts